java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -247,4 +247,12 @@ public void set_missing_value_and_returnOldValue_is_null(BaseClient client) {
         String data = client.set("another", ANOTHER_VALUE, options).get();
         assertNull(data);
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void config_reset_stat(BaseClient client) {
```

Comment:
I don't see any tests on the python client for `configResetStat`

Reply:
Feel free to add new 😉